websites/G/Gogoanime/dist/metadata.json

```diff
@@ -1,23 +1,26 @@
 {
   "author": {
-    "name": "PsychoPast",
+    "name": "Psycho",
     "id": "444779014888882177"
   },
+  "contributors": [{
+    "name": "Undo",
+    "id": "246660882803720193"
+  }],
   "service": "Gogoanime",
   "description": {
-    "en": "Watch anime online in English. You can watch free series and movies online with English subtitles."
+    "en": "Watch anime online in English. You can watch free series and movies online and English subtitle. Free download high quality anime."
   },
-  "url": [
-    "gogoanime.vc",
-    "gogoanime.ai"
-  ],
```

Comment:
you still need atleast 1 url (as fallback for if for some reason the regex fails)